AshleyLab/motrpac-frontend
# MOTR-120 Update Open Graph Image

**Key change**
- Replace the old MoTrPAC image with the new logo image for Open Graph

**How to test using localtunnel:**
- Install Localtunnel : `$ npm install -g localtunnel`
- Use the command line interface to request a tunnel to your local server: `$ lt -p 3000`
- You will receive a url, for example https://chilly-snake-95.localtunnel.me, that you can type it on Slack to see the preview image.

**Test Result:** 
- I tested by hosting it using localtunnel to check the updated preview image. The screenshot below is on Slack with the update image logo.
![open_graph](https://user-images.githubusercontent.com/14895583/61069487-3ed9fd80-a3c1-11e9-84a9-6aab8fd4714d.png)
- **Note**: In prod environment, we will need to test absolute URLs for images since the relative path on og:image meta tag will result in broken preview image.
